react-bootstrap/react-bootstrap.d.ts

```diff
@@ -190,6 +190,12 @@ declare module "react-bootstrap" {
         dialogComponent?: any; // TODO: Add more specific type
         enforceFocus?: boolean;
         keyboard?: boolean;
+        onEnter?: Function;
+        onEntered?: Function;
+        onEntering?: Function;
+        onExit?: Function;
+        onExited?: Function;
+        onExiting?: Function;
```

Comment:
prefer `() => any;`, or `(node: HTMLElement) => any;` based on the [source code](https://github.com/react-bootstrap/react-overlays/blob/master/src/Transition.js#L98-L114)